src/deployment/web.md

```diff
@@ -97,8 +64,8 @@ following structure:
 ```
 
 {{site.alert.note}}
-  The `canvaskit` directory and its contents are only present when the
-  CanvasKit renderer is selected—not when the HTML renderer is selected.
+The `canvaskit` directory and its contents are only present when the
+CanvasKit renderer is selected—not when the HTML renderer is selected.
```

Comment:
**issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).  Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
When you select the CanvaKit renderer, the `canvaskit` directory
and its contents appear. When you select the HTML renderer,
this directory doesn't appear.
```